adapters/deepintent/deepintent.go

```diff
@@ -0,0 +1,142 @@
+package deepintent
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/mxmCherry/openrtb"
+	"github.com/prebid/prebid-server/adapters"
+	"github.com/prebid/prebid-server/errortypes"
+	"github.com/prebid/prebid-server/openrtb_ext"
+)
+
+// DeepintentAdapter struct
+type DeepintentAdapter struct {
+	URI string
+}
+
+// NewDeepintentBidder Initializes the Bidder
+func NewDeepintentBidder(endpoint string) *DeepintentAdapter {
+	return &DeepintentAdapter{
+		URI: endpoint,
+	}
+}
+
+func (d *DeepintentAdapter) makeRequest(request *openrtb.BidRequest) (*adapters.RequestData, []error) {
```

Comment:
Please change the make of this method to better separate it from the `MakeRequests` method,

Reply:
changed the name of the method